guides/plugins/plugins/testing/php-unit.md

```diff
@@ -196,72 +187,33 @@ cd ./../../../
 Now we can execute the `phpunit.sh` to run all tests of our plugin.
 
 ```text
-$ ./bin/phpunit.sh
+./bin/phpunit.sh
```

Comment:
do we really need this wrapper script? I would rather show the real phpunit command with the options. Feels weird to wrap it especially in the docs.

Reply:
I guess it was added to remove one point of error since passing the wrong path will break the execution. But to understand the command better, adding the full command makes sense :+1: